packages/editor/src/components/widgets/combobox/Combobox.tsx

```diff
@@ -35,6 +36,7 @@ const Combobox = <T extends ComboboxItem>({
   onChange,
   macro,
   browserTypes,
+  asCell,
```

Comment:
I think I understand why it is needed. It is because the cell and the combobox currently both "save" the value if the focus is removed from it and the new value need to go through both of them.

What I don't like is this parameter name. If I see this combobox I would not understand it's purpose until I read the code. Nether would I understand what this parameter changes if I would use this combobox somewhere. A parameter should be named after the purpose what it's doing. In this case it defines when the onChange is triggered, directly after each change or after a blur/focus loose. So it should be called something like 'directUpdate' or 'directOnChange'... Maybe you have a better name in mind.

Reply:
Ah yes, thats a good point. I think I will rename the parameter to "updateOnInputChange" 👍